src/main/java/mate/academy/rickandmorty/model/MovieCharacter.java

```diff
@@ -0,0 +1,25 @@
+package mate.academy.rickandmorty.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+
+@Entity
+@Data
+@Table(name = "api_characters")
+public class MovieCharacter {
```

Comment:
```suggestion
public class Character {
```
You have MovieCharacter and Character widely used in both forms, which causes confusion, please unify